### DISCUSSION TITLE
Config Folder

### DISCUSSION THREAD

**Discussion by Diseum** [MinecraftServerControl/mscs#337] [Q&A]
When adding mods to an MSCS controlled server where would you find the config folder for the mods? I have tried searching in most of the folders I thought they would be in but I am unable to find the config files for the mods.

**Comment by zanix** [comment]
If I recall correctly, it should be in the same directory as the server files.
So in the case of Forge, this would be in `/opt/mscs/server/forge-xxxx/`
